docs/_guide/properties.md

```diff
@@ -26,77 +26,134 @@ LitElement manages your declared properties and their corresponding attributes.
 
 </div>
 
-### Property options
-
-A property declaration is an object in the following format:
-
-```
-{ optionName1: optionValue1, optionName2: optionValue2, ... }
-```
-
-The following options are available:
 
-* `converter`: [Convert between properties and attributes](#conversion).
-* `type`: [Use LitElement's default attribute converter](#conversion-type).
-* `attribute`: [Configure observed attributes](#observed-attributes).
-* `reflect`: [Configure reflected attributes](#reflected-attributes).
-* `noAccessor`: Whether to set up a default [property accessor](#accessors).
-* `hasChanged`: Specify what constitutes a [property change](#haschanged).
-
-All property declaration options can be specified in a static properties getter, or with TypeScript decorators.
 
 ## Declare properties {#declare}
 
-Declare your element's properties by implementing a static `properties` getter, or by using decorators:
+Declare your element's properties by implementing a static `properties` field, or by using decorators:
 
-```js
-// properties getter
+_Properties field_
+
+<div class="language-js">
```

Comment:
I was adding internal formatting (see the `<var>` tags below) to do this more as an abstract syntax section, rather than an example. I tried to see if I could get it to do code highlighting as well, but that didn't work. But I'm leaving the tags like this so it gets the same default formatting as highlighted snippets.

Reply:
ok, thanks